### ISSUE TITLE
Texas IAMOnline account for ReportStream

### ISSUE THREAD

**Issue by TaneaY** [CDCgov/prime-reportstream#14770]
## Problem statement

Texas IAMOnline password for account is expiring tomorrow on 6/18 3 days from 6/15 notification received.

For further help or if this email was received in error, please contact the Help Desk at 512-438-4720 or 855-435-7181 (toll-free), 7 a.m. to 7 p.m. CST, Monday–Friday.

## What you need to know

Current account is under Mo's email address

## Acceptance criteria

- Move IAMOnline account for Texas from Mo to the general ReportStream inbox -- THIS IS NO LONGER NEEDED
- [x] Move all auth to Victor's new Key-Pair account
- [ ] Remove Mo's old account
- [ ] Ensure new public/Private key pair is stored in a secure location that is accessible by future onboarding engineers

## To do
Recommendation from Mo "any personal account TX set up for me should be retired and everything should point only to the ReportStream inbox."
Update account details(username and password) with Texas IAMOnline Portal
Advise if and or how Support will triage this account in the future and where the changes will be housed.

**Comment by chris-kuryak** [comment]
Question to ask:
- Is the username for this account tied to the public/private key pair that we use to send data in staging or production?

**Comment by chris-kuryak** [comment]
Victor to email his existing contact at Texas (Nadja) to determine if Mo's account is needed to send data to TX in staging or production.

**Comment by chris-kuryak** [comment]
I emailed Nadja on 6/20.
Her response on 6/21 indicates that this issue is not affecting our connection with Texas.

We will wait until Victor returns from vacation (July 1) to determine next steps.

**Comment by chris-kuryak** [comment]
Meeting set up for Thurs morning to disposition

**Comment by chris-kuryak** [comment]
@victor-chaparro to respond to Nadje's email

**Comment by chris-kuryak** [comment]
Victor responded and latest is that TX with the public key pair to migrate the accounts.
Nadja has not responded, but they said they would create a ticket with the Globalscape support team. 

Waiting on response from TX now.

**Comment by chris-kuryak** [comment]
Victor is moving all creds over to the new Public/Private key pair account. We can remove Mo's old account.
AC's updated.

**Comment by chris-kuryak** [comment]
Victor to let Nadja know to remove Mo's old account.

**Comment by TaneaY** [comment]
Closing with Nadja comment as Mo's account is inactive and will close automatically at 90 days inactivity.

Hi Victor,
Looks like the account IDIELR011 is the active submitter, see below:
Do you want me to have acct idielr007 put in disabled state? Typically we archive accounts with 6+ months of inactivity.
Note, the acct idielr007 will disable automatically if inactive for 90 days.  
Thanks!
